### PR TITLE
[Website] Improve reuqest library link

### PR DESCRIPTION
**Pull request by qas612820704**
#195

### CODE REVIEW

File: templates/layout.html [hunk 1]
```diff
@@ -122,7 +122,7 @@
       </div>
     </footer>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js" integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ=" crossorigin="anonymous"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/2.4.2/lodash.min.js" integrity="sha256-rKk2QnJsnOCsuS8oSzkedgInNJbYmA09J0w26nVBpss=" crossorigin="anonymous" defer></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.4/lodash.min.js" integrity="sha256-8E6QUcFg1KTnpEU8TFGhpTGHw5fJqB9vCms3OhAYLqw=" crossorigin="anonymous"></script>
```
Comment (PeterDaveHello) [inline templates/layout.html]: Can we still use `defer` attribute here?

Comment (qas612820704) [reply templates/layout.html]: Oh! That's be my mistake. I will add it back.

**Comment by qas612820704** [comment]
OK!

**Comment by PeterDaveHello** [comment]
@qas612820704 so we suppose to have another commit above the current existing two commits which only upgrade the lodash?

**Comment by qas612820704** [comment]
@PeterDaveHello , you mean I should split `upgrade loadsh` by an independent commit ?

**Comment by PeterDaveHello** [comment]
@qas612820704 yes please

**Comment by qas612820704** [comment]
@PeterDaveHello , OK!

**Comment by PeterDaveHello** [comment]
Awesome!

**Pull request merged**